catalog/ui/search-ui/standard/src/main/webapp/js/model/user.js

```diff
@@ -94,14 +94,15 @@ define([
             };
         },
         parse: function(resp) {
-            resp.label = 'Type: ' + resp.type;
-            if (resp.layer) {
-                resp.label += ' Layer: ' + resp.layer;
+            var layer = _.clone(resp);
```

Comment:
This parse method looks like something that should go in the serializeData method of a view.